nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -1939,18 +2047,6 @@ mod test {
         assert_eq!(collection2.cabooses.len(), coll_counts.cabooses);
         assert_eq!(collection2.rot_pages.len(), coll_counts.rot_pages);
 
-        // Check that we get an error on the limit being reached for
```

Comment:
How about adding a test that uses a batch size of 1 and makes sure we get the same result for some reasonably representative collection?  (That way we know the pagination stuff is working.)

Reply:
Hmm, if we follow your advice from above about not making the batch size
configurable, we'd have to factor out a private function that takes a batch size
parameter...do you think it's worth doing that just for testing this?

Reply:
Yeah, I think that's easy enough to do and worthwhile.  I've been burned by a surprising number of pagination bugs in the past!